### DISCUSSION TITLE
Can regular v128 user.js be used on Firefox ESR v128?

### DISCUSSION THREAD

**Discussion by Raven-621** [yokoffing/Betterfox#307] [Q&A] [answered]
https://www.mozilla.org/en-US/firefox/128.0esr/releasenotes/

Since a new version of ESR came out and it's at least the same version number, can the regular Betterfox be used on the ESR version of Firefox? or is it better to just wait for the ESR version update of Betterfox?

**Comment by yokoffing** [comment, marked as answer]
Off the top of my head, yes. I'll make a separate 128 ESR branch soon and link it in the repo.

**Reply by yokoffing** [reply]
https://github.com/yokoffing/Betterfox/tree/esr128
